modules/images/image-loading-optimization/detection.php

```diff
@@ -45,7 +48,9 @@ function ilo_get_detection_script( string $slug, array $needed_minimum_viewport_
 		'urlMetricsNonce'             => ilo_get_url_metrics_storage_nonce( $slug ),
 		'neededMinimumViewportWidths' => $needed_minimum_viewport_widths,
 		'storageLockTTL'              => ilo_get_url_metric_storage_lock_ttl(),
+		'webVitalsLibrarySrc'         => $web_vitals_lib_src,
```

Comment:
```suggestion
		'webVitalsLibrarySrc'         => esc_url( $web_vitals_lib_src ),
```

Escaping needed here?

Reply:
yes, given https://github.com/WordPress/wordpress-develop/blob/89c53f4db76bf4a0f541516f8032dac690af8a3a/src/wp-includes/functions.php#L1119-L1121